Index: visit-2.7.0/src/visit_vtk/full/vtkCSGCell.h
===================================================================
--- visit-2.7.0.orig/src/visit_vtk/full/vtkCSGCell.h	2014-01-22 12:13:22.000000000 +0000
+++ visit-2.7.0/src/visit_vtk/full/vtkCSGCell.h	2014-01-22 12:13:22.000000000 +0000
@@ -49,6 +49,7 @@
 #include <visit_vtk_exports.h>
 
 #include "vtkCell.h"
+#include "vtkPointLocator.h"
 
 class vtkPointLocator;
 
