Index: visit-2.8.1/src/CMake/FindHDF4.cmake
===================================================================
--- visit-2.8.1.orig/src/CMake/FindHDF4.cmake
+++ visit-2.8.1/src/CMake/FindHDF4.cmake
@@ -56,6 +56,8 @@ IF (WIN32)
   else()
       SET_UP_THIRD_PARTY(HDF4 lib include hdf mfhdf xdr)
   endif()
-ELSE (WIN32)
-  SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
+ELSEIF(DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdfalt dfalt)
+ELSE (DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
 ENDIF (WIN32)
