Index: visit-2.7.0/src/CMake/FindHDF4.cmake
===================================================================
--- visit-2.7.0.orig/src/CMake/FindHDF4.cmake	2014-01-22 12:03:50.000000000 +0000
+++ visit-2.7.0/src/CMake/FindHDF4.cmake	2014-01-22 12:05:00.000000000 +0000
@@ -46,6 +46,8 @@
 
 IF (WIN32)
   SET_UP_THIRD_PARTY(HDF4 lib include hdfdll mfhdfdll xdrdll)
-ELSE (WIN32)
-  SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
+ELSEIF(DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdfalt dfalt)
+ELSE (DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
 ENDIF (WIN32)
