Index: visit2.2.2/src/CMake/FindHDF4.cmake
===================================================================
--- visit2.2.2.orig/src/CMake/FindHDF4.cmake	2011-04-19 12:08:50.000000000 +0100
+++ visit2.2.2/src/CMake/FindHDF4.cmake	2011-04-19 13:56:00.000000000 +0100
@@ -52,6 +52,8 @@
       SET(HDF4_INCLUDE_DIR "${HDF4_INCLUDE_DIR}/WIN32;${HDF4_INCLUDE_DIR}" CACHE PATH "HDF4 include directory" FORCE)
     ENDIF (CMAKE_CL_64)
   ENDIF (HDF4_FOUND)
-ELSE (WIN32)
-  SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
+ELSEIF(DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdfalt dfalt)
+ELSE (DEBIAN)
+    SET_UP_THIRD_PARTY(HDF4 lib include mfhdf df)
 ENDIF (WIN32)
